src/config-hardware/audio-mic.md

```diff
@@ -0,0 +1,67 @@
+# Audio & Microphone Settings
+
+This section describes selection and configuration of audio input and output devices. Set system volume, volume levels per application, output devices, and input devices in `Settings` ➞ `Sound`.
+
+Pop!\_OS 22.04 uses the [PipeWire](https://github.com/pop-os/pipewire) server to handle audio and video streams. Previous versions of Pop!\_OS use [PulseAudio](https://www.freedesktop.org/wiki/Software/PulseAudio/).
+
+---
+
+## PipeWire
+
+Pop!\_OS 22.04 and above uses the PipeWire sound server. A sound server is a low-level service that controls communication of audio streams between the software and hardware.
+
+## Setting Volume Levels
+
+Drag the slider under `Volume Levels` right or left to increase or decrease the volume level.
+
+![Sound Settings](/images/audio-mic/sound-settings.png)
+
+### Over-Amplification
+
+`Over-Amplification` allows you to raise the volume over 100%. This can result in loss of audio quality. As a best practice, it is recommended to increase the volume in the application's individual volume settings.
```

Comment:
This description is just copying what's in the Settings copy (and in the screenshot.) If you want to say something about it, I would say something about why it's useful rather than just its downsides: `This allows the volume of media or applications that are too quiet to be digitally boosted.`